### DISCUSSION TITLE
Remove old sample images in J4

### DISCUSSION THREAD

**Discussion by chmst** [joomla/joomla-cms#32744] [2. Cold Features]
Joomla is shipped with sampledata images. 

images/sampledata/fruitshop
images/sampledata/parks
images/headers

in images/banners are rather "antique" banners from a time where Joomla was young. 

What's your meaning about removing these  images and folders in J4?

We do not need to remove them during migration,  maybe users have built demo sites using these images, but in fresh installations they are not longer needed.

**Comment by rdeutz** [comment]
If you remove them from the repo the will not be in new installations but stay in old. Not sure if they are used in some of our sample data, if so you have to remove that also.

**Comment by HLeithner** [comment]
I think this is related https://github.com/joomla/joomla-cms/issues/31992

**Reply by brianteeman** [reply]
they are used in the testing sample data plugin BUT that is broken already anyway

**Reply by chmst** [reply]
The links to images have been removed, so this does not make a difference.

**Reply by brianteeman** [reply]
great

**Reply by dgrammatiko** [reply]
FWIW the sampledata plugin **should** copy over these files, they shouldn't be the responsibility of Joomla's installer (ie images folder should be empty). The right way is demonstrated https://github.com/dgrammatiko/sloth-pkg/tree/main/plg_sampledata

**Reply by chmst** [reply]
@brianteeman lol - I had not seen that you have already removed them, except the headers 
https://github.com/joomla/joomla-cms/pull/32009

**Comment by richard67** [comment]
I agree with removing them for new installations but not removing them on updates, i.e. not add them to the list of files to be deleted in script.php. That means they have to be added to the exclude list in @wilsonge 's PR #25559 and in @dgrammatiko 's current development on https://github.com/dgrammatiko/jdiff for a new tool for that purpose after a future PR which will remove them from the GitHub sources will be merged.

**Reply by laoneo** [reply]
Agree 👍
